### PR TITLE
[14.0][ADD] account_invoice_refund_reason_skip_anglo_saxon

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

Depends:

- [x] https://github.com/OCA/account-invoicing/pull/1273

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]
Code review LGTM

File: account_invoice_refund_reason_skip_anglo_saxon/tests/test_account_invoice_refund_reason_skip_anglo_saxon.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            self.env["account.move.reversal"].with_context(
+                active_ids=invoice.ids,
+                active_model="account.move",
+            )
+        )
+        reversal_form.reason_id = self.refund_reason_skip_anglo_saxon
+        reversal_wizard = reversal_form.save()
+        action = reversal_wizard.reverse_moves()
+        refund = self.env["account.move"].browse(action.get("res_id"))
+        # FIXME: until Open PR get cherry-pick
```
Comment (AaronHForgeFlow) [inline account_invoice_refund_reason_skip_anglo_saxon/tests/test_account_invoice_refund_reason_skip_anglo_saxon.py:92]: Can you reference the Pull request in the same comment? for easier check

**Review by AaronHForgeFlow — APPROVED** [review]
code review LG!

File: account_invoice_refund_reason_skip_anglo_saxon/tests/test_account_invoice_refund_reason_skip_anglo_saxon.py (hunk 1 above)
Comment (ChrisOForgeFlow) [reply account_invoice_refund_reason_skip_anglo_saxon/tests/test_account_invoice_refund_reason_skip_anglo_saxon.py:92]: Now is not necessary

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**